DataModelHelloWorld/Cpp/SimpleIntroClientLibrary/SimpleIntroClientLibrary.vcxproj

```diff
@@ -172,20 +176,38 @@
     <ClCompile Include="SimpleIntroExtension.cpp" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="amd64\dbgcore.dll" />
+    <None Include="amd64\dbgeng.dll" />
+    <None Include="amd64\dbghelp.dll" />
+    <None Include="amd64\dbgmodel.dll" />
+    <None Include="arm64\dbgcore.dll" />

```

Comment:
><None Include="arm64\dbgcore.dll" /> [](http://example.com/codeflow?start=4&length=36)

If you reference ARM64 binaries, perhaps you should also add the ARM64 build flavor?

Then again, that might make the project unloadable unless you install the ARM64 toolset in VS.

Don't know. Just a thought.

Reply:
This is automatic from the Microsoft.Debugging.Platform.DbgEng NuGET.  We'll have to update that...